mappings/net/minecraft/datafixer/fix/LeavesFix.mapping

```diff
@@ -1,4 +1,22 @@
 CLASS net/minecraft/class_1191 net/minecraft/datafixer/fix/LeavesFix
+	FIELD field_5686 logBlocks Ljava/util/Set;
+	FIELD field_5688 leafMap Lit/unimi/dsi/fastutil/objects/Object2IntMap;
```

Comment:
static final fields should use upper camel case, like `LEAVES_MAP`

Reply:
Whoops, missed that

Reply:
(same goes for `LOG_BLOCKS` etc)